feature/policy_forwarding/GUE/Outer_IPv4/decapsulation/otg_tests/Inner_MPLS/README.md

```diff
@@ -0,0 +1,104 @@
+# PF-1.5: Interface based MPLSoGUE Decapsulation to IPv4 tunnel
+
+## Summary
+
+This is to test the the functionality of policy-based forwarding (PF) to decapsulate Generic UDP Encapsulation (GUE) traffic. These tests verify the use case of MPLSoGUE to IPv4 GUE tuennel. The tests are meant for `Tunnel Interface` or `Policy Based` implementation of IPv4 GUE tunnel. The tests validate that the DUT performs the following action.
+
+ - DUT is a transient node to forward MPLSoGUE traffic.
+ - DUT is a target node to decapsulate IPv4GUE and forward inner MPLSoGUE to the destination. 
+
+
+## Testbed type
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+* Create the following connections:
+* DUT has ingress and egress port connected to the ATE.
+  
+```mermaid
+graph LR; 
+A[ATE:Port1] --Ingress--> B[Port1:DUT:Port2];B --Egress--> C[Port2:ATE];
+```
+
+*  ATE Port 1: Generates GUE-encapsulated traffic with various inner (original) destinations.
```

Comment:
Please expand on the specification of "various" -- what is the expected number of 5-tuples and what fields do they vary in?